entity-framework/core/performance/advanced-performance-topics.md

```diff
@@ -295,14 +295,17 @@ Compiled models have some limitations:
 
 * [Global query filters are not supported](https://github.com/dotnet/efcore/issues/24897).
 * [Lazy loading and change-tracking proxies are not supported](https://github.com/dotnet/efcore/issues/24902).
-* Value converters that reference private methods are not supported. Make referenced methods public or internal instead.
 * [The model must be manually synchronized by regenerating it any time the model definition or configuration change](https://github.com/dotnet/efcore/issues/24894).
 * Custom IModelCacheKeyFactory implementations are not supported. However, you can compile multiple models and load the appropriate one as needed.
 
 Because of these limitations, you should only use compiled models if your EF Core startup time is too slow. Compiling small models is typically not worth it.
 
 If supporting any of these features is critical to your success, then please vote for the appropriate issues linked above.
 
+### Handling compilation errors due to ambiguous type references
```

Comment:
Does this belong here in "advanced performance topics"?

Reply:
This page contains all the compiled model documentation that we have